src/components/ResourceCard/ResourceCard.js

```diff
@@ -37,7 +35,12 @@ const selectCardBody = (resource) => {
     // case 'Claims':
     //   return <ClaimCardBody fieldsData={fieldsData} />;
     case 'Encounter':
-      return <EncounterCardBody resource={resource} />;
+      return (
+        <EncounterCardBody
```

Comment:
so we are only associating `serviceProvider` with `Encounter`? I would think that every card body _could_ have a `serviceProvider`, but is only rendered if the `resource` actually has one.

What about "2-degree connections" like `Condition` > `Encounter` > `ServiceProvider` === `Condition` > `ServiceProvider`? Perhaps that can be a follow up PR...

Reply:
yes, a follow-up would be needed to handle these transitive relationships.